### ISSUE TITLE
Generic.WhiteSpace.ScopeIndent shows invalid error when scope opener indented inside inline HTML

### ISSUE THREAD

**Issue by elcamlost** [squizlabs/PHP_CodeSniffer#2167]
phpcbf is failed to fix following code

```php
	<?if($arParams["DISPLAY_BOTTOM_PAGER"] != "N") echo $arResult["NAV_STRING"]; ?>

<?else:?>
	<?ShowNote(GetMessage("SEARCH_NOTHING_TO_FOUND"));?>
<?endif;?>
```

the result is

```
php phpcbf.phar --encoding=utf-8 --extensions=php --standard=PSR1,PSR2  example.php

PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
/Users/camlost/cetera/stmc/tmp/t.php                  FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 0 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------

Time: 101ms; Memory: 4Mb
```
This is with PHP 7.1.16, phpcbf 3.3.1

The main problem here is not "phpcbf can't fix code" but the error message. It gives no clue what string causes the problem.

**Comment by elcamlost** [comment]
I still got the same error. Maybe this screenshot will help to sheed some light on the issue 
![screenshot](https://user-images.githubusercontent.com/5188593/46583154-c376a480-ca5a-11e8-9281-d7a540ec1651.jpg)

**Comment by gsherwood** [comment]
There must be something more that is happening with your file:
```
$ cat example.php
    <?if($arParams["DISPLAY_BOTTOM_PAGER"] != "N") echo $arResult["NAV_STRING"]; ?>

<?else:?>
    <?ShowNote(GetMessage("SEARCH_NOTHING_TO_FOUND"));?>
<?endif;?>
$ php phpcbf.phar --version
PHP_CodeSniffer version 3.3.2 (stable) by Squiz (http://www.squiz.net)
$ php phpcbf.phar --encoding=utf-8 --extensions=php --standard=PSR1,PSR2 example.php

No fixable errors were found

Time: 43ms; Memory: 4Mb
```

> My original point was "it's inconvenient to use second tool to understand why the first tool failed to do the task". According to your comment above I conclude it's a design limitation for phpcbf and recommended way is running phpcs to better understand why and where phpcbf can't fix error automatically.

PHPCS is a reporting tool while PHPCBF is a fixing tool. They actually share the exact same codebase and can do the exact same things. But the tool you select focuses the functionality to one of the intended goals (reporting or fixing).

So yes, you need to use PHPCS to see what errors the file actually has first. I get no errors, which is the first sign that we aren't testing the same thing. Then you can continue using PHPCS to do a dry-run of the fixing process and see what is going wrong. If you post that information, I might be able to figure out if there is a bug here that I can fix.

If you're not worried about the bug and just wanted to comment about the fact there is no verbose debug output for PHPCBF, hopefully I've answered that question now.

**Comment by exussum12** [comment]
I get the exact same as here

FILE: /tmp/test.php
-----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-----------------------------------------------------------------------
 3 | ERROR | [x] Line indented incorrectly; expected 0 spaces, found 4
-----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-----------------------------------------------------------------------


PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
/tmp/test.php                                         FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 0 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------



[test.php.txt](https://github.com/squizlabs/PHP_CodeSniffer/files/2470325/test.php.txt)

**Comment by gsherwood** [comment]
@exussum12 I think I figured out what is going on - the short open tags from the original code sample were being ignored for my configuration. The sample file you provided allowed me to replicate the original issue. Thanks a lot for posting that.

**Comment by gsherwood** [comment]
I've now committed a fix for this, which will be in 3.4.0
